### PR TITLE
Adding a file for raleigh nc

### LINKED ISSUE #4: GitHubTeacher's recommendations for things to do/eat
## Adding Raleigh, NC recommendations:
- [ ] Places to eat
- [ ] Things to do

### PR DESCRIPTION
**Pull request by githubteacher**
## ⚠️ WIP ⚠️

Closes #4

### CODE REVIEW

File: githubteacher-raleigh.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Recommendations for Raleigh, NC
+
+### Places to Eat
+
```
Comment (githubteacher) [inline githubteacher-raleigh.md:4]: I really like Pizza and Tacos

File: githubteacher-raleigh.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+### Things to Do
```
Comment (githubteacher) [inline githubteacher-raleigh.md:8]: Any recommendations for people who like to hike?

**Review by githubteacher — COMMENTED** [review]
Submitting a review of my own PR

**Review by brianamarie — APPROVED** [review]
🚢

**Pull request merged**